docs/en/operations/settings/settings.md

```diff
@@ -840,19 +840,19 @@ Default value: `uniqExact`.
 
 ## skip_unavailable_shards {#settings-skip_unavailable_shards}
 
-Enables or disables silent skipping of:
+Enables or disables silently skipping:
 
-- Node, if its name cannot be resolved through DNS.
+- Nodes, if their name can't be resolved through DNS.
```

Comment:
It's only a special case of unavailable replica.
And these replicas are always skipped.